modules/http/src/main/java/org/glassfish/grizzly/http/ChunkedTransferEncoding.java

```diff
@@ -49,6 +49,9 @@ public final class ChunkedTransferEncoding implements TransferEncoding {
     private static final byte[] LAST_CHUNK_CRLF_BYTES = "0\r\n".getBytes(ASCII_CHARSET);
     private static final int[] DEC = HexUtils.getDecBytes();
 
+    public static final String STRICT_CHUNKED_TRANSFER_CODING_LINE_TERMINATOR_RFC_9112 = "org.glassfish.grizzly.http.STRICT_CHUNKED_TRANSFER_CODING_LINE_TERMINATOR_RFC_9112";
+    private static final boolean isStrictChunkedTransferCodingLineTerminatorSet = Boolean.parseBoolean(System.getProperty(STRICT_CHUNKED_TRANSFER_CODING_LINE_TERMINATOR_RFC_9112));
+
     private final int maxHeadersSize;
 
     public ChunkedTransferEncoding(final int maxHeadersSize) {
```

Comment:
If you can, would you create an overloaded constructor that accepts `Properties` and attempts to read the config from there before it reads the config from System properties?

And to complement it, would you also create overloaded constructors with a `Properties` argument in `HttpServerFilter`, `HttpCodecFilter`, and `HttpClientFilter` that pass it to ChunkedTransferEncoding?

If not, I'll try to do that in another PR, but it would really help. With this, it would be then possible to configure this option in constructor, individually for each server config, rather than using a system property for all listeners. Later we would integrate it in GlassFish to allow setting this property via standard GlassFish config for each listenerrather than via a global system property.

Reply:
@OndroMih I applied your suggestion.

While modifying, I thought a little about the priority of `Properties` and system properties and how values ​​are applied. The current implementation is to apply only one of the two, and to give priority to the `Properties` setting.

PS) Another way is to merge or override the settings.